functions/go/set-label/LabelTransformer_test.go

```diff
@@ -0,0 +1,195 @@
+package main
```

Comment:
Are these tests generated from the tests in kustomize or manually written?

Reply:
I added them manually to avoid importing bunch of kustomize internal test utilities.